### PR TITLE
Boilerplate spec feedback related to TCK, JCP and acronyms

### PR DESCRIPTION
**Pull request by spericas**
Feedback provided on boilerplate spec after initial PR was merged.

Signed-off-by: Santiago Pericas-Geertsen <santiago.pericasgeertsen@oracle.com>

### CODE REVIEW

File: jaxrs-spec/src/main/asciidoc/boilerplate.adoc [hunk 1]
```diff
@@ -16,24 +16,20 @@
...
-The acronym JAX-RS is reused with permission of Oracle and either refers to "Jakarta RESTful Web Services" or "Java™ API for RESTful Web Services", dependending on the context.
+*Jakarta RESTful Web Services* provides a specification document, TCK and foundational API to develop web services
+following the Representational State Transfer (REST) architectural pattern.
 
 
 === TCK
 
-NOTE: At time of writing, there is no standalone TCK for *Jakarta RESTful Web Services*. To proof compliance with this specification, pass the Jakarta EE Platform TCK instead.
+The TCK for this release is available as a ZIP file from Eclipse at the following location:
+http://download.eclipse.org/ee4j/jakartaee-tck/8.0.1/release/standalone-tcks/jaxrstck-2.1_latest.zip
```
[the remark this answers is not in the document]
Comment (spericas) [reply jaxrs-spec/src/main/asciidoc/boilerplate.adoc]: Link updated.

File: jaxrs-spec/src/main/asciidoc/boilerplate.adoc [hunk 2]
```diff
@@ -16,24 +16,20 @@
 
 == Boilerplate
 
-WARNING: This document contains just boilerplate information. It explicitly is *not* a complete specification. In particular, it misses all normative text. To learn about compliance with this specification, refer to "JAX-RS: Java™ API for RESTful Web Services" (Version 2.1, Final Release July 13, 2017).
+WARNING: This document contains just boilerplate information. It explicitly is *not* a complete specification.
```
Comment (bshannon) [inline jaxrs-spec/src/main/asciidoc/boilerplate.adoc]: Why do you think this spec in particular needs this warning but none of the other specs do?

File: jaxrs-spec/src/main/asciidoc/boilerplate.adoc [hunk 3]
```diff
@@ -16,24 +16,20 @@
...
-The acronym JAX-RS is reused with permission of Oracle and either refers to "Jakarta RESTful Web Services" or "Java™ API for RESTful Web Services", dependending on the context.
+*Jakarta RESTful Web Services* provides a specification document, TCK and foundational API to develop web services
+following the Representational State Transfer (REST) architectural pattern.
 
 
 === TCK
 
-NOTE: At time of writing, there is no standalone TCK for *Jakarta RESTful Web Services*. To proof compliance with this specification, pass the Jakarta EE Platform TCK instead.
+The TCK for this release is available as a ZIP file from Eclipse at the following location:
+https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8/
```
Comment (bshannon) [inline jaxrs-spec/src/main/asciidoc/boilerplate.adoc]: There's no need to add a reference to the TCK in this spec.
This link does *not* point to the EFTL-licensed TCK that is required
for compatibility claims so it doesn't provide much value here.

File: jaxrs-spec/src/main/asciidoc/boilerplate.adoc (hunk 2 above)
Comment (mkarg) [reply jaxrs-spec/src/main/asciidoc/boilerplate.adoc]: I do not think that actually. Actually I do think that *all* Boilerplate Specs should countain that warning so readers understand that *essential* information is missing.

**Comment by spericas** [comment]
> You can also get rid of the wombat-1.png file.

It has been removed

**Comment by spericas** [comment]
> As this is the sole change in this commit, and as the changed line was introduced in the same PR, I would suggest to squash this before merge.

Sure, will do

File: jaxrs-spec/src/main/asciidoc/boilerplate.adoc (hunk 2 above)
Comment (spericas) [reply jaxrs-spec/src/main/asciidoc/boilerplate.adoc]: Although I understand what @mkarg is saying, looking at other boilerplate specs (e.g., JSONB and JSONP), they don't include any of this information. In the interest of time, we should follow the same approach and continue this discussion at a later time.

**Pull request merged**